src/Microsoft.ML.FastTree/GamTrainer.cs

```diff
@@ -1028,6 +1030,118 @@ void ICanSaveSummary.SaveSummary(TextWriter writer, RoleMappedSchema schema)
             Runtime.Numeric.VectorUtils.SparsifyNormalize(ref contributions, top, bottom, normalize);
         }
 
+        public void SaveAsIni(TextWriter writer, RoleMappedSchema schema, ICalibrator calibrator = null)
+        {
+            Host.CheckValue(writer, nameof(writer));
+            Host.CheckValue(schema, nameof(schema));
+            Host.CheckValueOrNull(calibrator);
+
+            var ensemble = new TreeEnsemble();
+
+            for (int featureIndex = 0; featureIndex < _numFeatures; featureIndex++)
+            {
+                var effects = _binEffects[featureIndex];
+                var binThresholds = _binUpperBounds[featureIndex];
+
+                Host.Assert(effects.Length == binThresholds.Length);
+                var numLeaves = effects.Length;
+                var numInternalNodes = numLeaves - 1;
+
+                var splitFeatures = Enumerable.Repeat(featureIndex, numInternalNodes).ToArray();
+                var (treeThresholds, lteChild, gtChild) = CreateBalancedTree(numInternalNodes, binThresholds);
+                var tree = CreateRegressionTree(numLeaves, splitFeatures, treeThresholds, lteChild, gtChild, effects);
+                ensemble.AddTree(tree);
+            }
+
+            // Tried adding the intercept as the bias term in the final ini aggregator,
+            // but that didn't seem to have any effects during testing.
+            // Adding the intercept as a dummy tree with the output values being the model intercept,
```

Comment:
This is a good comment for review, but for the final check-in, just keep this line.